TradeItIosTicketSDK2/TradeItAccountSelectionViewController.swift

```diff
@@ -29,16 +29,17 @@ class TradeItAccountSelectionViewController: UIViewController, TradeItAccountSel
                                                                onRefreshComplete: (withLinkedBrokers: [TradeItLinkedBroker]?) -> Void) {
         // TODO: Need to think about how not to have to wrap every linked broker action in a call to authenticate
         self.linkedBrokerManager.authenticateAll(
-            onSecurityQuestion: { (securityQuestion: TradeItSecurityQuestionResult, onAnswerSecurityQuestion: (String) -> Void, onCancelSecurityQuestion: () -> Void) in
+            onSecurityQuestion: { securityQuestion, onAnswerSecurityQuestion, onCancelSecurityQuestion in
                 self.alertManager.show(
                     securityQuestion: securityQuestion,
                     onViewController: self,
                     onAnswerSecurityQuestion: onAnswerSecurityQuestion,
                     onCancelSecurityQuestion: onCancelSecurityQuestion
                 )
             },
-            onFailure:  { (tradeItErrorResult: TradeItErrorResult, linkedBroker: TradeItLinkedBroker) -> Void in
+            onFailure:  { tradeItErrorResult, linkedBroker in
                 self.alertManager.show(tradeItErrorResult: tradeItErrorResult, onViewController: self, withLinkedBroker: linkedBroker, onFinished: {
+                    // QUESTION: is this just going to re-run authentication for all linked brokers again if one failed?
                         onRefreshComplete(withLinkedBrokers: self.linkedBrokerManager.getAllEnabledLinkedBrokers())
```

Comment:
@guillaumedebavelaere is this something intentional or am I reading it incorrectly? If the first linked broker fails, then this is going to call `onRefreshComplete` even though there might be other accounts authenticating still.